app/webpack.development.ts

```diff
@@ -58,6 +58,9 @@ const rendererConfig = merge({}, common.renderer, config, {
       },
     ],
   },
+  infrastructureLogging: {
+    level: 'error',
+  },
```

Comment:
This used to be the `logLevel` property from `devMiddleware` (https://github.com/desktop/desktop/pull/13894/files#diff-404124522cf8054ccbb64a424ba4e5de30e9c059c05b8cfaab3ad4339633f6c9L55)